src/ransacflow/train.py

```diff
@@ -50,7 +50,7 @@ def computeLossMatchability(network, I, indexRoll, grid, maskMargin, args, ssim)
 
     ## Reconstruction Loss 3 channels 
     IWarp = F.grid_sample(I, final)
-    lossLr =  ssim(IWarp, I[indexRoll], matchCycle)
+    lossLr =  SSIM(IWarp, I[indexRoll], args.margin)
```

Comment:
I think `margin` and `window` might be two different arguments?

Reply:
The window size is fixed in def __init__(self, window_size = 11). 
And the margin is input as m (=88), where I crop the center of 48 by img1[..., m:-m, m:-m], img2[..., m:-m, m:-m] in the def forward () function.